packages/pipeline/src/Stage.tsx

```diff
@@ -0,0 +1,208 @@
+import React, { forwardRef, ReactNode, Ref, ReactElement } from 'react';
+import PropTypes from 'prop-types';
+import mergeRefs from 'react-merge-refs';
+import { useInView } from 'react-intersection-observer';
+
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import {
+  getRootStyle,
+  getChildStyle,
+  getChevronStyle,
+  Size,
+  Variant,
+  layout,
+  colors,
+} from './styles';
+
+interface StateForStyles {
+  size: Size;
+  variant: Variant;
+}
+
+interface StageProps {
+  /**
+   * Content that will appear inside of the Pipeline component.
+   **/
+  children: ReactNode;
+
+  /**
+   * Classname applied to Pipeline content container.
```

Comment:
Think you mean className applied to the Stage content container, ditto with above (children prop)

Reply:
Whoops, copy/paste typo. Will update.

Reply:
Resolved in 4fb3d4b